github/provider.go

```diff
@@ -92,6 +99,8 @@ func init() {
 			"without verifying the TLS certificate.",
 
 		"individual": "Whether to run outside an organization.",
+
+		"anonymous": "Whether to authenticate without a token.",
```

Comment:
My problem with "whether or not to do X" descriptions of booleans is the cognitive hurdle it takes to determine whether `true` or `false` leads to X actually happening. `Set to true to authenticate without a token` avoids this shortcoming, as an example.